pallets/rc-migrator/src/proxy.md

```diff
@@ -25,8 +25,34 @@ The [delay of a ProxyDefinition](https://github.com/paritytech/polkadot-sdk/blob
 ## Storage: Announcements
 
 The [Announcements](https://github.com/paritytech/polkadot-sdk/blob/7c5224cb01710d0c14c87bf3463cc79e49b3e7b5/substrate/frame/proxy/src/lib.rs#L581-L592) storage maps proxy account IDs to [Accouncement](https://github.com/paritytech/polkadot-sdk/blob/7c5224cb01710d0c14c87bf3463cc79e49b3e7b5/substrate/frame/proxy/src/lib.rs#L80-L89). Since an announcement contains a call hash, we cannot translate them for the same reason as with the Multisigs; the preimage of the hash would be either undecodable, decode to something else (security issue) or accidentally decode to the same thing.  
-
 We therefore do not migrate the announcements.
+
+## Pure Proxy Migration
+
+Pure proxies are [derived](https://github.com/paritytech/polkadot-sdk/blob/4f7a93885e1a35ec60178d3b4f1e59a7df3d85f5/substrate/frame/proxy/src/lib.rs#L790-L806) in the form of `hash(BlockNumber, ParentAcc, ProxyType)`. The important part here is the `ProxyType` enum. The [relay chain](https://github.com/polkadot-fellows/runtimes/blob/main/relay/polkadot/constants/src/lib.rs#L177-L189) and [Asset Hub](https://github.com/polkadot-fellows/runtimes/blob/main/system-parachains/asset-hubs/asset-hub-polkadot/src/lib.rs#L456-L471) have the following variants:
+
+| Index | Relay Chain | Asset Hub | Index Good | Unused on AH | Verdict |
+|-------|-------------|-----------|-----------------|-------------|---------|
+| 0 | Any | Any | ✅ | ❌ | NoOp |
+| 1 | NonTransfer | NonTransfer | ✅ | ✅ | NoOp |
+| 2 | Governance | CancelProxy | ❌ | ✅ | Disable on AH |
+| 3 | Staking | Assets | ❌ | ✅ | Disable on AH |
+| 4 | - | AssetOwner | ✅ | ❌ | NoOp |
+| 5 | - | AssetManager | ✅ | ✅ | NoOp |
+| 6 | CancelProxy | Collator | ❌ | ✅ | Disable on AH |
+| 7 | Auction | TBD | ✅ | ✅ | NoOp |
+| 8 | NominationPools | TBD | ✅ | ✅ | NoOp |
+| 9 | NominationParaRegistration | TBD | ✅ | ✅ | NoOp |
+
+There is good news and bad news here. The good news is that there is only one account with an `AssetOwner` proxy and that has an index that is unused by the Relay Chain. All other proxies are using `Any`. This means that the case that one proxy imposters as another proxy - by exploiting the colliding enum indices - should not happen. Such an attack could otherwise be devastating, since it could allow one proxy to irrevocably kill another proxy. For example; a `Governance` proxy on the Relay chain could then delete a `CancelProxy` on the asset hub. Deletion (via `Proxy::kill_pure`) is the "only" thing that can be done by this attack.
```

Comment:
`kill_proxy` still can be done only by the `spawner`, even if there a collision, it is still the same `spawner`